manifests/init.pp

```diff
@@ -746,10 +768,13 @@
   }
 
   # export each node's ssh key
-  @@sshkey { $::fqdn :
```

Comment:
I believe if you don't have exported resources enabled with PuppetDB, this is just silently skipped. Great having the ability to not export it though, especially if using PuppetDB, to keep unwanted bits out of your database.